spec/api-browser-window-spec.ts

```diff
@@ -5338,6 +5338,48 @@ describe('BrowserWindow module', () => {
       });
     });
 
+    describe('when fullscreen state is changed', () => {
```

Comment:
`isMenuBarVisible` and `setMenuBarVisible` are Windows/Linux only - you'll need to change this `describe` to

```
ifdescribe(process.platform !== 'darwin')('when fullscreen state is changed', () => {
```